drivers/npmx/npmx_shell.c

```diff
@@ -840,17 +859,22 @@ static int buck_gpio_set(const struct shell *shell, size_t argc, char **argv,
 	}
 
 	int err = 0;
-	uint8_t buck_indx = (uint8_t)shell_strtoul(argv[1], 0, &err);
+	uint8_t buck_indx = (uint8_t)(shell_strtoul(argv[1], 0, &err) <= UINT8_MAX ?
+					      shell_strtoul(argv[1], 0, &err) :
+					      UINT8_MAX);
 	int gpio_indx = shell_strtol(argv[2], 0, &err);
 
+	if ((shell_strtoul(argv[3], 0, &err) != 0) && (shell_strtoul(argv[3], 0, &err) != 1)) {
+		shell_error(shell, "Error: GPIO inv state has to be a bool value.");
+		return 0;
+	}
+
 	npmx_buck_gpio_config_t gpio_config = {
 		.inverted = !!shell_strtoul(argv[3], 0, &err),
```

Comment:
I think that the result of `shell_strtoul(argv[3])` could be saved as a variable, like `buck_indx` or `gpio_indx` so that it would not have to be called three times.

Reply:
Yup, it will be done but currently I have a fierce fight with CI so it is a temporary solution